pkg/console/subresource/deployment/deployment.go

```diff
@@ -79,6 +95,7 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 	meta.Annotations[configMapResourceVersionAnnotation] = cm.GetResourceVersion()
 	meta.Annotations[serviceCAConfigMapResourceVersionAnnotation] = serviceCAConfigMap.GetResourceVersion()
 	meta.Annotations[secretResourceVersionAnnotation] = sec.GetResourceVersion()
+	meta.Annotations[routeResourceVersionAnnotation] = rt.GetResourceVersion()
```

Comment:
Is this needed? The public URL is in the config map. Is there another time we need to force a rollout when the route changes?

Reply:
If our availability flickers, I'd like to be able to see clearly what resource changed, via its resource version, on the deployment.  This is just to give us a clue should routing ever flutter.

Reply:
But it will force unnecessary rollouts whenever the route status changes. This could force several unnecessary rollouts on initial install. It would be better to simply log the route status.

Reply:
Good point. 😅 
I'll eliminate this and just log the changes.